controllers/src/test/scala/cromwell/pipeline/controller/AkkaHttpClientTest.scala

```diff
@@ -0,0 +1,80 @@
+package cromwell.pipeline.controller
+
+import akka.event.LoggingAdapter
+import akka.http.scaladsl.model.{ HttpRequest, HttpResponse }
+import akka.http.scaladsl.settings.ConnectionPoolSettings
+import akka.http.scaladsl.{ HttpExt, HttpsConnectionContext }
+import cromwell.pipeline.service.Response
+import org.mockito.Matchers.any
+import org.mockito.Mockito.when
+import org.scalatest.{ AsyncWordSpec, BeforeAndAfterAll, Matchers }
+import org.scalatestplus.mockito.MockitoSugar
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class AkkaHttpClientTest extends AsyncWordSpec with Matchers with MockitoSugar with BeforeAndAfterAll {
+  implicit val ec: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
+
+  val mockHttp: HttpExt = mock[HttpExt]
+  val cromwellResponse: Response = Response(200, "", Map())
+
+  class TestAkkaHttpClient extends AkkaHttpClient {
+    override def http: HttpExt = mockHttp
+  }
```

Comment:
We could use WireMock instead of mocking it in this way

Reply:
Didn't find this dependency in project. Should I add it and stub server for verify client response?